Comments/ClientApp/src/components/LeadInfo/LeadInfo.js

```diff
@@ -0,0 +1,60 @@
+﻿import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import Cookies from "js-cookie";
+import { withHistory } from "../HistoryContext/HistoryContext";
+import BreadCrumbsWithRouter, { Breadcrumbs } from "../Breadcrumbs/Breadcrumbs";
+import { load } from "../../data/loader";
+import { queryStringToObject } from "../../helpers/utils";
+
+export const LeadInfo = () => {
+	const emailAddress = "DITApplicationSupport@nice.org.uk"
+	const emailSubject = encodeURIComponent("New commenting lead request");
+	const emailBody = encodeURIComponent("Dear Sir/Madam\r\rMy name:\rI want to be commenting lead for\r\rKind regards");
+
+	return (
+		<div className="container">
+
+			<Breadcrumbs links={[{label:"Home",url:"/"},{label:"All consultations", url:"/guidance/inconsulation"}]} />
+
+			<div className="grid">
+				<div data-g="12">
+					<h2>What is a commenting lead?</h2>
+					<p>Commenting leads are representatives of an organisation who collate feedback on NICE consultations from colleagues within their organisation. They are also responsible for submitting their organisation's consultation response to NICE.</p>
+					<p>Leads are now able to do this online using our commenting system, but will require a set of permissions which can be assigned on request.</p>
+				</div>
+			</div>
```

Comment:
```suggestion
```

Reply:
Think it's better to nest the below grid inside of the col-12 from above. Again, sorry for providing wrong advice on this. Also made a change below concerning this and the nesting of divs.